javapeople.yaml

```
@@ -195,6 +195,10 @@
 - name: "GraalVM"
   twitter: "@graalvm"
   fediverse: "@graalvm@mastodon.online"
+- name: Greg Chabala
```

```suggestion
- name: "Greg Chabala"
```
Thank you for this as well, all looks good except missing quotations here.